config/xsl/evaluation/sections/rules.xsl

```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
+
+<xsl:template name="rules">
```

Comment:
It is a bit surprising that this template produces two sections, Rules and Rule Checks. How about splitting it into two templates files rules.xsl and rule-checks.xsl? Might involve moving the multicell template to utils.xsl.

Reply:
Done.